ixwebsocket/IXSocket.cpp

```diff
@@ -73,7 +73,7 @@ namespace ix
 
         struct timeval timeout;
         timeout.tv_sec = timeoutMs / 1000;
-        timeout.tv_usec = (timeoutMs < 1000) ? 0 : 1000 * (timeoutMs % 1000);
+        timeout.tv_usec = 1000 * (timeoutMs % 1000);
```

Comment:
That's good / I wonder how we ended up with that.

Reply:
Me too :+1: )